module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -274,6 +274,12 @@ public function getMergedConfig(string $key = '', bool $flatten = false): array
             $currentTheme = $allThemeInfo[$currentTheme]['extends'];
         }
 
+        if (empty($merged)) {
```

Comment:
As noted elsewhere, I don't think we need/want to change the behavior of getMergedConfig here. Note that getMergedConfig already has in-progress revisions in #2543. We should try to avoid creating conflicts with that PR if we can (and if there are problems here related to merged configs, we should see if that PR solves them).